utils/use_require.js

```diff
@@ -39,15 +39,14 @@ const no_transform_files = new Set([
     path.join(paths.app, 'error-handler.js'),
 ]);
 
-no_copy_files.forEach((file) => no_transform_files.add(file));
+no_copy_files.forEach(file => no_transform_files.add(file));
 
 // util.promisify requires Node.js 8.x, so we have our own
 function promisify(original) {
-    return function () {
-        const obj = this;
+    return () => {
         const args = Array.prototype.slice.call(arguments);
         return new Promise((resolve, reject) => {
-            original.apply(obj, args.concat((err, value) => {
+            original.apply(this, args.concat((err, value) => {
```

Comment:
This does not have the same behaviour as the previous code. I don't think you can use an arrow function here.

Reply:
Why not?

As in the other comment above, `const obj = this;` and we are not doing any binding or calling the function on any other object so obj will always be equals to this.

Also, what do you mean about not being able to use an arrow function?